Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -349,17 +349,35 @@ def get_user(user, properties):
 
 def list_users_command():
     properties = demisto.args().get('properties', 'id,displayName,jobTitle,mobilePhone,mail')
-    users_data = list_users(properties)
-
+    next_page = demisto.args().get('next_page', None)
+    users_data, result_next_page = list_users(properties, next_page)
     users_readable, users_outputs = parse_outputs(users_data)
     human_readable = tableToMarkdown(name='All Graph Users', t=users_readable, removeNull=True)
+
     outputs = {'MSGraphUser(val.ID == obj.ID)': users_outputs}
+
+    if result_next_page:
+        human_readable = tableToMarkdown(name='All Graph Users', t=users_readable, removeNull=True,
```

Comment:
i would do
metadata="To get further results, enter this to the next_page parameter:\n" + str(result_next_page)

and then remove line 355 and put the human readable assignment in line 363 after the if

Reply:
ok